client/webserver/locales/parse_test.go

```diff
@@ -0,0 +1,210 @@
+    </form>
+
+    {{- /* AUTHORIZE IMPORT ACCOUNT */ -}}
+    <form class="d-hide" id="authorizeAccountImportForm">
+      {{template "authorizeAccountImportForm" .}}
+    </form>
+
+    {{- /* NEW WALLET */ -}}
+    <form class="d-hide" id="newWalletForm">
+      {{template "newWalletForm" }}
+    </form>
+
+    {{- /* CHANGE APP PASSWORD */ -}}
+    <form class="d-hide" id="changeAppPWForm">
+      {{template "changeAppPWForm"}}
+    </form>
+
+    {{- /* EXPORT SEED AUTHORIZATION */ -}}
+    <form class="d-hide" id="exportSeedAuth">
+      <div class="form-closer hoverbg"><span class="ico-cross"></span></div>
+      <div class="py-1 text-center position-relative fs22 sans-light">
+        [[[Export Seed]]]
+      </div>
+      <div class="fs15 text-start mt-2">
+        [[[pw_for_seed]]]
+      </div>
+      <div class="text-start mt-2">
+        <label for="exportSeedPW" class="ps-1 mb-1">[[[Password]]]</label>
+        <input type="password" class="form-control select" id="exportSeedPW" autocomplete="current-password">
+      </div>
+      <div class="d-flex justify-content-end mt-4">
+        <button id="exportSeedSubmit" type="button" class="justify-content-center fs15 bg2 selected">[[[Show Me]]]</button>
+      </div>
+      <div class="fs15 pt-3 text-center d-hide errcolor text-break" id="exportSeedErr"></div>
+    </form>
+
+    {{- /* SEED DISPLAY */ -}}
+    <form class="d-hide" id="authorizeSeedDisplay">
+      <div class="form-closer hoverbg"><span class="ico-cross"></span></div>
+      <div class="fs18 sans-light text-center mb-2">[[[dont_share]]]</div>
+      <div id="seedDiv"></div>
+    </form>
+
+    {{- /* SYNC AND BALANCE FORM */ -}}
+    <form class="d-hide" id="walletWait">
+      {{template "waitingForWalletForm"}}
+    </form>
+
+  </div>
+
+</div>
+{{template "bottom"}}
+{{end}}
+`)
+	wantTokens := []string{
+		"[[[Dark Mode]]]",
+		"[[[Show pop-up notifications]]]",
+		"[[[fiat_exchange_rate_msg]]]",
+		"[[[fiat_exchange_rate_sources]]]",
+		"[[[registered dexes]]]",
+		"[[[simultaneous_servers_msg]]]",
+		"[[[Add a DEX]]]",
+		"[[[Import Account]]]",
+		"[[[Change App Password]]]",
+		"[[[seed_implore_msg]]]",
+		"[[[View Application Seed]]]",
+		"[[[Build ID]]]",
+		"[[[Export Seed]]]",
+		"[[[pw_for_seed]]]",
+		"[[[Password]]]",
+		"[[[Show Me]]]",
+		"[[[dont_share]]]",
+	}
+	sort.Slice(wantTokens, func(i, j int) bool {
+		return wantTokens[i] < wantTokens[j]
+	})
+	wantKeys := []string{
+		"Dark Mode",
+		"Show pop-up notifications",
+		"fiat_exchange_rate_msg",
+		"fiat_exchange_rate_sources",
+		"registered dexes",
+		"simultaneous_servers_msg",
+		"Add a DEX",
+		"Import Account",
+		"Change App Password",
+		"seed_implore_msg",
+		"View Application Seed",
+		"Build ID",
+		"Export Seed",
+		"pw_for_seed",
+		"Password",
+		"Show Me",
+		"dont_share",
+	}
+	sort.Slice(wantKeys, func(i, j int) bool {
+		return wantKeys[i] < wantKeys[j]
+	})
+
+	got := Tokens(tmplFile)
```

Comment:
I'm not so sure the settings html template is the best test.  There's are dozens of lines of irrelevant text, plus the tokens being tested don't include all the characters in `keyRegExp`.  How about a more concise and contrived template, with more keys to test the full intended char set?

Reply:
I used this html template because it is a ready-made real-world example, I don't think for this particular test it's an issue to have all this text included,

added a harder pattern as per your suggestion, 

but if you have some specific pattern in mind - I can update to that, of course